dcc-portal-ui/app/scripts/index.js

```diff
@@ -102,4 +102,5 @@ require('./pancancer/js/pancancer.js');
 require('./translations/js/translations.js');
 require('./entitysetupload/js/entitysetupload.js');
 require('./404/js/404.js');
+require('./304/js/304.js');
```

Comment:
Turns out this needs to be 301:

http://stackoverflow.com/questions/1393280/http-redirect-301-permanent-vs-302-temporary